app.py

```diff
@@ -19,6 +19,40 @@
 
 file_mapping = {} # store orginal file name pair with converted file name
 
+from transformers import pipeline
+import torch
+import argostranslate.package 
+import argostranslate.translate
+
+# helper function for transcription
+def transcribe_audio(audio_file_path):
+    device = "cuda:0" if torch.cuda.is_available() else "cpu"
+    
+    pipe = pipeline(
+        "automatic-speech-recognition",
+        model="openai/whisper-large-v3-turbo",
+        device=device,
+    )
+    
+    result = pipe(audio_file_path)
+    
+    return result["text"]
+
+# helper function for translation
+def translate_text(text, from_code, to_code):
+    argostranslate.package.update_package_index()
+    available_packages = argostranslate.package.get_available_packages()
+    package_to_install = next(
+        filter(
+            lambda x: x.from_code == from_code and x.to_code == to_code,
+            available_packages
+        )
+    )
+    argostranslate.package.install_from_path(package_to_install.download())
+    
+    translated = argostranslate.translate.translate(text, from_code, to_code)
+    return translated
+
```

Comment:
Might be in here causing segmentation fault

Reply:
I tried updating the dockerfile, I think it may have been because of compatibility issues and the dockerfile not including the correct libraries. Try rebuilding the docker and see if it works.

<img width="497" alt="2024-12-20 10_30_36-usage md - FormatConverter - Visual Studio Code" src="https://github.com/user-attachments/assets/e2db7f88-dc5b-43ff-b263-3c50cb45034a" />

Reply:
<img width="858" alt="Screenshot 2024-12-20 at 5 13 51 PM" src="https://github.com/user-attachments/assets/cb0a67ac-080e-43b4-8457-f283f0fd9681" />

Are those "\u00bfc" things expected?

Reply:
yup thats because of spanish using accent marks like è

Reply:
Nice！LGTM